### PR TITLE
feat(umi-ui): add mac address

### PR DESCRIPTION
**Pull request by ikobe**
chore: 调整 tasks ui 部分细节

##### Checklist

- [ ] `npm test` passes
- [ ] tests are included
- [ ] documentation is changed or added
- [ ] commit message follows commit guidelines

##### Description of change

- any feature?
- close https://github.com/umijs/umi/ISSUE_URL

### CODE REVIEW

File: packages/umi-ui/src/scripts.ts [hunk 1]
```diff
@@ -35,6 +51,7 @@ const ga = `
   gtag('js', new Date());
 
   gtag('config', 'UA-145890626-1');
```
Comment (ycjcl868) [inline packages/umi-ui/src/scripts.ts]:
```suggestion
   gtag('config', 'UA-145890626-1', {
    'user_id': '{{ roleId }}'
  });
```

File: packages/umi-ui/src/scripts.ts [hunk 2]
```diff
@@ -35,6 +51,7 @@ const ga = `
...
+  gtag('set', {'user_id': '{{ roleId }}'});
```
Comment (ycjcl868) [inline packages/umi-ui/src/scripts.ts]:
```suggestion

```

File: packages/umi-ui/src/scripts.ts (hunk 1 above)
Comment (ikobe) [reply packages/umi-ui/src/scripts.ts]: fixed

File: packages/umi-ui/src/scripts.ts [hunk 3]
```diff
@@ -1,3 +1,18 @@
+import macaddress from 'macaddress';
+
+let macId = '';
```
Comment (ycjcl868) [inline packages/umi-ui/src/scripts.ts]: 为啥要放外面

**Pull request merged**